cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules.yaml

```diff
@@ -349,4 +349,4 @@ spec:
         (ceph_pool_stored_raw * on (pool_id) group_left(name)ceph_pool_metadata) / ((ceph_pool_quota_bytes * on (pool_id) group_left(name)ceph_pool_metadata) > 0) > 0.90
       for: 1m
       labels:
-        severity: critical
\ No newline at end of file
+        severity: critical
```

Comment:
plz, Leave an empty line at the end of the file.

Reply:
@parth-gr Thanks for the review. Is it good now?